app/src/main/java/org/fossasia/pslab/experimentsetup/NFETOutputCharacteristicsExperiment.java

```diff
@@ -198,9 +198,9 @@ protected Void doInBackground(Float... params) {
             float voltage = params[0];
             scienceLab.setPV1(voltage);
             float readVoltage = (float) scienceLab.getVoltage("CH1", 10);
-            voltageAxis.add(readVoltage);
-            float resistance = 560;
-            currentAxis.add((voltage - readVoltage) / resistance);
+            voltageAxis.add(Math.abs(readVoltage - 3));
+            float resistance = 56;
```

Comment:
Could you tell a little about these parameters changes like resistance 56 instead of 560 and adding modified voltage to axis-values